### ISSUE TITLE
Driver Error: unable to load driver, class std:exception, "unremovable service mismatch"

### ISSUE THREAD

**Issue by GoogleCodeExporter** [soul4soul/peerblock#202]
```
What steps will reproduce the problem?
1.install/copy+paste PeerBlock(either install version or zip version)
2.run peerblock.exe
3.error message pops up(attached)

What is the expected output? What do you see instead?

The GUI of PeerBlock. I see an error message window.

What version of PeerBlock are you using? On what operating system? 32- or
64-bit?

PeerBlock 1.0+ (r202) BETA
PeerBlock 1.0 (r181) Win32

OS: Windows XP 32-bit - Build:[2600], SP:[3.0]

Please provide any additional information below.  Make sure to attach
peerblock.log and/or any screenshots that would help explain your problem.


```

Original issue reported on code.google.com by `konglong.1984` on 12 Nov 2009 at 9:05
- Merged into: #205

Attachments:
- [peerblock.log](https://storage.googleapis.com/google-code-attachments/peerblock/issue-202/comment-0/peerblock.log)
- [error.png](https://storage.googleapis.com/google-code-attachments/peerblock/issue-202/comment-0/error.png)

**Comment by GoogleCodeExporter** [comment]
```
This is actually being caused by Issue #205 - I'll leave this one open though 
(with
an updated summary/description) to help people find it.  A workaround can be 
found in
this forum-post: (http://forums.peerblock.com/read.php?3,2674)


```

Original comment by `peerbloc...@gmail.com` on 1 Dec 2009 at 3:00
- Changed title: **Driver Error: unable to load driver, class std:exception, "unremovable service mismatch"**
- Changed state: **Accepted**
- Added labels: **Milestone-1.1**, **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
Original comment by `peerbloc...@gmail.com` on 1 Dec 2009 at 4:35
- Added labels: **App-Crash**

**Comment by GoogleCodeExporter** [comment]
```
This bug should be resolved by the Issue #205 fix.  After testing the next Beta
Release, if you still end up encountering this error please comment here and 
we'll
reopen this one for investigation.
```

Original comment by `peerbloc...@gmail.com` on 1 Mar 2010 at 6:21
- Changed state: **Duplicate**
